mongodbatlas/data_source_mongodbatlas_project_api_key.go

```diff
@@ -39,6 +39,26 @@ func dataSourceMongoDBAtlasProjectAPIKey() *schema.Resource {
 				Elem: &schema.Schema{
 					Type: schema.TypeString,
 				},
+				Deprecated: fmt.Sprintf(DeprecationMessageParameterToResource, "v1.12.0", "project_assignment"),
```

Comment:
Just for consistency, this flag can also be added in project api keys data source

Reply:
upadated